msom/src/main/webapp/resources/js/processingsystems.js

```diff
@@ -10,8 +11,52 @@
                    cache : false,
                    success : function(data) {
                        $(".simulation-container").html(data);
+                       checkHiddenElements();
                    }
                 });
             }, 1000);
         }
     });
+
+    function toggleDetails(self)
+    {   
+        $(self.parentNode).find('.tasks-list').toggle(); 
+        if(isHidden.includes(self.parentElement.children[2].className)){
+            self.innerHTML = 'hide details';
+        isHidden.splice( $.inArray(self.parentElement.children[2].className, isHidden), 1 );      
+    }
+        else{
+          isHidden.push(self.parentElement.children[2].className);
+       
+      }
+    }
+
+     function checkHiddenElements(){
+         for (var i = 0;i<isHidden.length;i++)
+         {
+             $("[class*='"+isHidden[i]+"']").parent().children()[1].innerHTML = 'show details';
+             $("[class*='"+isHidden[i]+"']").hide(); 
+         }
+     }
+     
+    $('.details-toogle').hover(function() {
```

Comment:
to mi nie chce dzialac... ja bym po prostu napisala na stronie `<p style="cursor:pointer">`